WalletWasabi.Gui/Config.cs

```diff
@@ -70,6 +70,10 @@ public Config(string filePath) : base(filePath)
 		[JsonProperty(PropertyName = "UseTor", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public bool UseTor { get; internal set; }
 
+		[DefaultValue(false)]
+		[JsonProperty(PropertyName = "CloseTor", DefaultValueHandling = DefaultValueHandling.Populate)]
```

Comment:
Possibly `TerminateTorOnExit`? It's longer but it's self-explaining.

Reply:
Good idea